clang/include/clang/Driver/Options.td

```diff
@@ -3785,7 +3785,14 @@ def ftarget_export_symbols : Flag<["-"], "ftarget-export-symbols">,
 def fno_target_export_symbols : Flag<["-"], "fno-target-export-symbols">,
   Visibility<[ClangOption, CLOption, DXCOption]>;
 def : Flag<["-"], "fsycl-rdc">, Visibility<[ClangOption, CLOption, DXCOption]>, Alias<fgpu_rdc>;
-def : Flag<["-"], "fno-sycl-rdc">, Visibility<[ClangOption, CLOption, DXCOption]>, Alias<fno_gpu_rdc>;
+def : Flag<["-"], "fno-sycl-rdc">,
+  Visibility<[ClangOption, CLOption, DXCOption]>, Alias<fno_gpu_rdc>,
+  HelpText<"Generate relocatable device code during SYCL offload target "
+           "compilation. Use of ‘-fno-sycl-rdc’ in combination with ‘-c’ will "
```

Comment:
Also on the same thread, what happens if someone does `-fno-sycl-rdc -c -fsycl-targets=spir64,spir64_gen`. If they pass the object/archive to the final device link stage will it correctly extract the two device code files (one final, one not) and run device linking on the spir one but not run it on the spir64_gen one? Maybe we could add a test for this

Reply:
Agreed, I will add a test to reflect the difference in behavior between AOT and JIT when using `-c -fno-sycl-rdc`